GliaWidgets/Sources/View/Chat/ChatView.swift

```diff
@@ -404,11 +412,8 @@ extension ChatView {
                 imageUrl: imageUrl
             )
             return .gvaQuickReply(view)
-        case let .gvaGallery(_, gallery):
-            // Temporary, since UI hasn't been implemented
-            let textView = UITextView()
-            textView.text = "Gallery: \(gallery.type.rawValue)"
-            return .gvaGallery(textView)
+        case let .gvaGallery(message, gallery, showImage, imageUrl):
```

Comment:
Perhaps it would be good to disable the SwiftLint rule that is triggered on this method, as I don't see how we can make it smaller.

Reply:
Fixed 4285cdde0a5c765ee09b36d30c2a36d0bfc1b777